pages/dkp/kommander/2.2/projects/project-quotas-limit-range/index.md

```diff
@@ -10,12 +10,17 @@ excerpt: Project Quotas and Limit Ranges can be set up to limit the number of re
 
 Project Quotas and Limit Ranges can be set up to limit the number of resources the Project team uses. Quotas and Limit Ranges are applied to all project clusters.
 
-![Project Quotas and Limit Ranges](../../img/project-quota.png)
-Project Quotas and Limit Ranges
+1. Select the workspace your project was created in from the workspace selection dropdown in the header.
```

Comment:
I am not sure if this is the best way to word it, but I think it might a bit clearer
```suggestion
1. Select the workspace that corresponds to the project you created from the selection dropdown in the header.
```